### ISSUE TITLE
Don't change URL for unsupported iOS versions

### ISSUE THREAD

**Issue by NiklasMerz** [GEDYSIntraWare/cordova-plugin-webview-proxy#21]
If this plugin just don't change the URL in the method `window.WebviewProxy.convertProxyUrl` on iOS 12, 11..., it will work.

_Originally posted by @JuHaYK in https://github.com/GEDYSIntraWare/cordova-plugin-webview-proxy/issues/20#issuecomment-925705299_

**Comment by JuHaYK** [comment]
Do you know how to detect that iOS version is less than 13?

**Comment by NiklasMerz** [comment]
If you have a look the WebktiProxy.js file of this plugin you can see that I implemented a check for the variable `window.CDV_ASSETS_URL`. One thing to make sure is that the app runs with a custom scheme. The plugin won't work when the assets are loaded from `file:`. 

Possibly there are some JS variables that only exists on new versions. Otherwise you could check WebKit versions or features. Not entirely sure. I might look for other options.

**Comment by JuHaYK** [comment]
Ok, if we don't change URL on unsupported iOS versions, we have again problems with cookie...
It's not possible to support iOS 11 and 12?

**Comment by NiklasMerz** [comment]
Yes I don't think it is possible to use this plugin on iOS 11 and 12. For these iOS versions you most likely need to find other workarounds for cookie issues.

**Comment by JuHaYK** [comment]
Can you explain the problem why it is not possible to support iOS 11 and 12? Can I perhaps help?
